### DISCUSSION TITLE
How to start? Whats a token? But tokens aren't allowed! EH?

### DISCUSSION THREAD

**Discussion by p-baum** [Rapptz/discord.py#6482] [Q&A]
I dont get this?  How do I log in?  I've created an app at discord. I have an ID and secret.  How do I login with it?

**Comment by Zomatree** [comment]
Go to the bot tab, click "create a bot account", then use the bot token.

**Comment by Nemika-Haj** [comment]
Read the Documentation, there are clear instructions on how to get started; https://discordpy.readthedocs.io/en/latest/index.html#getting-started
